src/components/OperatorClientMenu/OperatorClientMenu.jsx

```diff
@@ -53,7 +53,11 @@ OperatorClientMenu.propTypes = {
 
 const mapStateToProps = state => ({
   operatorFilter: state.chat.operatorFilter,
-  openChats: state.chat.chats.filter(chat => chat.open).length,
+  openChats: Object
```

Comment:
love this chaining yeeeeee

Reply:
Haha, since chats is now an `{}` instead of a `[]` I had to convert it back to an array to be compatible with some code, this being an example of one